handlers/handlers.go

```diff
@@ -10,3 +12,61 @@ func Ready(s *discordgo.Session, event *discordgo.Ready) {
 	fmt.Printf("event.Version: %v\n", event.Version)
 
 }
+
+func parseCommand(s string) (string, []string) {
+	var fields []string
+	var sb strings.Builder
+	var quote rune
+
+	// parse string, spliting on spaces tabs and newlines
+	// get everything in quotes
+	for _, c := range s {
+		if quote == 0 && (c == '\'' || c == '"' || c == '`') {
+			quote = c
+		} else if quote == c || (quote == 0 && (c == ' ' || c == '\n' || c == '\t')) {
+			quote = 0
+
+			field := sb.String()
+			if len(field) > 0 {
+				fields = append(fields, field)
+				sb.Reset()
+			}
+		} else {
+			sb.WriteRune(c)
+		}
+	}
+
+	// write out last string
+	field := sb.String()
+	if len(field) > 0 {
+		fields = append(fields, field)
+	}
+
+	return fields[0], fields[1:]
+}
+
+func MessageCreate(s *discordgo.Session, m *discordgo.MessageCreate) {
```

Comment:
Here's were we can start splitting our code into smaller, separate files. The handler functions can go into separate files in the same handlers package:

/handlers/ready.go
/handlers/message-create.go

If you wanted, we could even move the `parseCommand` function into a service package so it'll be reusable and easily testable.